api_app/analyzers_manager/observable_analyzers/phishstats.py

```diff
@@ -0,0 +1,56 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class PhishStats(ObservableAnalyzer):
+    """
+    Analyzer that uses PhishStats API to check if the observable is a phishing site.
+    """
+
+    base_url: str = "https://phishstats.info:2096/api/"
+
+    def __build_phishstats_url(self) -> str:
+        if self.observable_classification == self.ObservableTypes.IP:
+            endpoint = "phishing?_where=(ip,eq,{input})"
+            name = self.observable_name
+        elif self.observable_classification == self.ObservableTypes.URL:
+            endpoint = "phishing?_where=(url,like,~{input}~)&_sort=-id"
+            domain = self.observable_name.split("/")[2]
+            name = domain.split(".")[0]
```

Comment:
this is not needed, u can pass the observable_name as is to the input.

Reply:
For example:
Go to https://phishstats.info:2096/api/phishing?_where=(url,like,~https://github.com/TinToSer/bluekeep-exploit~)&_sort=-date

The page does takes 10-20 seconds to load but seems to work fine

Reply:
Yea It does take 10-20 sec to load in case of URL but doesn't works for domain

Reply:
After passing the observable_name as it is to input, I'm getting soft-time limit exceeding error, Should i increase soft-time limit back to 100 sec
(You directed me to decrease it to 30 from 100 in previous commit)

![Screenshot from 2021-11-21 19-48-03](https://user-images.githubusercontent.com/72073401/142765496-29c0cbc4-8df9-47e4-a609-6c5176d9f6d3.png)

Reply:
ah, my bad. Yes revert it back to 100 seconds then.